src/generic/splitter.cpp

```diff
@@ -520,6 +534,14 @@ void wxSplitterWindow::OnSize(wxSizeEvent& event)
     SizeWindows();
 }
 
+void wxSplitterWindow::OnDPIChanged(wxDPIChangedEvent& event)
+{
+    m_minimumPaneSize = event.ScaleX(m_minimumPaneSize);
+    m_sizeAfterDPIChange = true;
```

Comment:
Couldn't we just update `m_sashPosition` and `m_lastSize` here too to avoid adding this variable and extra adjustments in `OnSize()`?

Reply:
This indeed works fine.